sdcm/sct_config.py

```diff
@@ -169,1450 +186,1774 @@ def boolean(value):
-
-        dict(name="use_ldap_authentication", env="SCT_USE_LDAP_AUTHENTICATION", type=boolean,
-             help="When defined true, will create a docker container with LDAP and configure scylla.yaml to use it"),
-
-        dict(name="prepare_saslauthd", env="SCT_PREPARE_SASLAUTHD", type=boolean,
-             help="When defined true, will install and start saslauthd service"),
-
-        dict(name="ldap_server_type", env="SCT_LDAP_SERVER_TYPE", type=str,
-             help="This option indicates which server is going to be used for LDAP operations. [openldap, ms_ad]"),
-
-        dict(name="use_mgmt", env="SCT_USE_MGMT", type=boolean,
-             help="When define true, will install scylla management"),
-
-        dict(name="parallel_node_operations", env="SCT_PARALLEL_NODE_OPERATIONS", type=boolean,
-             help="When defined true, will run node operations in parallel. Supported operations: startup"),
-
-        dict(name="manager_prometheus_port", env="SCT_MANAGER_PROMETHEUS_PORT", type=int,
-             help="Port to be used by the manager to contact Prometheus"),
-
-        dict(name="target_scylla_mgmt_server_address", env="SCT_TARGET_SCYLLA_MGMT_SERVER_ADDRESS", type=str,
-             help="Url to the repo of scylla manager version used to upgrade the manager server"),
-
-        dict(name="target_scylla_mgmt_agent_address", env="SCT_TARGET_SCYLLA_MGMT_AGENT_ADDRESS", type=str,
-             help="Url to the repo of scylla manager version used to upgrade the manager agents"),
-
-        dict(name="update_db_packages", env="SCT_UPDATE_DB_PACKAGES", type=str,
-             help="""A local directory of rpms to install a custom version on top of
-                     the scylla installed (or from repo or from ami)"""),
-
-        dict(name="monitor_branch", env="SCT_MONITOR_BRANCH", type=str,
-             help="The port of scylla management"),
-
-        dict(name="db_type", env="SCT_DB_TYPE", type=str,
-             help="Db type to install into db nodes, scylla/cassandra"),
-
-        dict(name="user_prefix", env="SCT_USER_PREFIX", type=str,
-
-             help="the prefix of the name of the cloud instances, defaults to username"),
-
-        dict(name="ami_id_db_scylla_desc", env="SCT_AMI_ID_DB_SCYLLA_DESC", type=str,
-             help="version name to report stats to Elasticsearch and tagged on cloud instances"),
+class SCTConfiguration(BaseModel):
+    """
+    Class the hold the SCT configuration
+    """
 
-        dict(name="sct_public_ip", env="SCT_SCT_PUBLIC_IP", type=str,
-             help="""
-                Override the default hostname address of the sct test runner,
-                for the monitoring of the Nemesis.
-                can only work out of the box in AWS
-             """),
-        dict(name="sct_ngrok_name", env="SCT_NGROK_NAME", type=str,
-             help="""
+    multi_region_params: Annotated[list[str], IgnoredType] = Field(default=[], exclude=True)
```

Comment:
here is example of two possible ways of ignoring `IgnoredType` and `Field(exclude=True)`

both are a bit long, and not completely sure behave similarly